docs/clients/99_graphql/index.rst

```diff
@@ -36,6 +36,18 @@ Pointing your browser to ``http://127.0.0.1:8888/explore``
 will bring up a `GraphiQL`_ interface to EdgeDB. This interface can be
 used to try out queries and explore the GraphQL capabilities.
 
+
+Known Limitations
+=================
+
+- Due to the differences between EdgeQL and GraphQL syntax some
+  :eql:type:`enum <std::enum>` types cannot be properly reflected.
```

Comment:
Can you clarify this? What enum types? All of them?

Reply:
It's covered in the linked enum description, but I can include it here as well.

Reply:
You say "some enum types" -- what does "some" mean? Does the linked description explains what "some" mean in this sentence? Sorry, I just really can't parse this and hence potentially nitpicking.